### DISCUSSION TITLE
VS 17.8 performance issues while debugging Oqtane/Blazor

### DISCUSSION THREAD

**Discussion by W6HBR** [oqtane/oqtane.framework#3601] [General]
Is anyone else seeing issues with performance while debugging Oqtane modules (or Blazor in general)?  The browser UI is sluggish and updates that previously were instantaneous now take several seconds.  This seems to have started in VS 17.8.

Starting the app from the command line or publishing/running from IIS runs normal (fast).  I've only found a couple of posts on the VS dev community on this, but no resolution as of yet.  

Wondering if anyone else has experienced this and uncovered anything with it.

Related VS dev community posts:  
https://developercommunity.visualstudio.com/t/Since-upgrading-to-Visual-Studio-2022-17/10518022?sort=newest
https://developercommunity.visualstudio.com/t/Visula-Studio-Become-very-slow/10537721?q=intellisense&sort=newest

**Comment by thabaum** [comment]
It does seem to lag out with connection icons appearing now and again I never noticed before when browsing, but I am testing areas I have not worked with yet and not sure if changes at times are causing the delay in responsiveness.  Is this what you are referring to?
